### ISSUE TITLE
Previous item is pasted when using Albert

### ISSUE THREAD

**Issue by phillip-white-sociomantic** [hluk/CopyQ#948]
This has come up in Albert https://github.com/albertlauncher/albert/issues/658

When using Albert to paste the last item, the second last item is pasted instead.

@hluk I think you already have a fix for the issue?

**Comment by hluk** [comment]
Yup, should be fixed in 9d94036e44d1b3811209890a6a6de7d10d3539da. It should work fine if you use `copyq` build from recent master branch.
